src/layouts/EditHomepage.jsx

```diff
@@ -33,62 +33,62 @@ import DeleteWarningModal from '../components/DeleteWarningModal';
 const RADIX_PARSE_INT = 10;
 
 // Section constructors
-const ResourcesSectionConstructor = () => ({
+const ResourcesSectionConstructor = (isErrorConstructor) => ({
   resources: {
-    title: '',
-    subtitle: '',
-    button: '',
+    title: isErrorConstructor ? '' : 'Resources Section Title',
+    subtitle: isErrorConstructor ? '' : 'Resources Section Subtitle',
+    button: isErrorConstructor ? '' : 'Resources Button Name',
   },
 });
 
-const InfobarSectionConstructor = () => ({
+const InfobarSectionConstructor = (isErrorConstructor) => ({
   infobar: {
-    title: '',
-    subtitle: '',
-    description: '',
-    button: '',
-    url: '',
+    title: isErrorConstructor ? '' : 'Infobar Title',
+    subtitle: isErrorConstructor ? '' : 'Infobar Subtitle',
+    description: isErrorConstructor ? '' : 'Infobar description',
+    button: isErrorConstructor ? '' : 'Button Text',
+    url: isErrorConstructor ? '' : '/faq/',
   },
 });
 
-const InfopicSectionConstructor = () => ({
+const InfopicSectionConstructor = (isErrorConstructor) => ({
   infopic: {
-    title: '',
-    subtitle: '',
-    description: '',
-    button: '',
-    url: '',
-    imageUrl: '',
-    imageAlt: '',
+    title: isErrorConstructor ? '' : 'Infopic Title',
+    subtitle: isErrorConstructor ? '' : 'Infopic Subtitle',
+    description: isErrorConstructor ? '' : 'Infopic description',
+    button: isErrorConstructor ? '' : 'Button Text',
+    url: isErrorConstructor ? '' : '/faq/',
+    image: isErrorConstructor ? '' : '/image/',
```

Comment:
`/faq` and `/image` might not necessarily exist on the user's repo. I'm wondering if we can replace these with default text that is more descriptive, such as `No URL specified` and `No image selected`, but I understand it would be a bit weird since in the future we might want to validate these fields. However I did test it locally and the site builds even with nonsense values for `url` and `image`. What are your thoughts?

Reply:
After discussing with designers, we want to:
- Allow default value of `url` to be an empty string, and set the input element's placeholder value to be `Insert permalink or external URL`
- Allow default value of `image` to be an empty string

Reply:
Modified after discussion with designers. Url is now left blank with different placeholder text, and image is left blank as it is handled by the modal. Resolved in https://github.com/isomerpages/isomercms-frontend/pull/172/commits/56b6b3f0ac39b706900c743c667387e01b184e97